example/pubspec.yaml

```diff
@@ -4,17 +4,17 @@ version: 0.0.1
 publish_to: none
 
 environment:
-  sdk: ">=3.4.0 <4.0.0"
+  sdk: ^3.5.0
```

Comment:
I know that Kevin Moore likes it when people use this constraint for the sdk but is that also going to be our way forward?

Reply:
It should be, I raised an issue in Flutter some time ago about it and is now the default when using `flutter create`. It is also documented as a good practice in the official Dart documentation.

See https://github.com/flutter/flutter/pull/150920